components/tools/OmeroPy/src/omero/cli.py

```diff
@@ -1567,6 +1568,8 @@ def __call__(self, arg):
             for id in parts[1].split(","):
                 if "-" in id:
                     needsForce = True
+                    warnings.warn("This force flag to delete range is deprecated as\
```

Comment:
I think there's still an element of surprise here since the user may or may not have passed the flag:

|             | --dry-run          | --force            | (no flag)                                                                                                            |
|-------------|--------------------|--------------------|----------------------------------------------------------------------------------------------------------------------|
| OMERO 5.3.1 | dry-run            | delete             | dry run                                                                                                              |
| OMERO 5.3.2 | No warning needed? | No warning needed? | warning("Using '--dry-run'. Future versions will switch to '--force'. Explicitly set the parameter for portability") |
| OMERO5.4    | dry-run            | delete             | delete                                                                                                               |